src/Searchlight/Parsing/StringConstants.cs

```diff
@@ -128,5 +128,15 @@ public class StringConstants
         /// Used for identifying the separator in a list
         /// </summary>
         public static readonly string COMMA = ",";
+
+        /// <summary>
+        /// Used as shorthand for typing today's date
+        /// </summary>
+        public static readonly Dictionary<string, DateTime> DefinedDates = new Dictionary<string, DateTime>
+        {
+            {"TODAY", DateTime.Today},
```

Comment:
Hmm.  My concern here is, this value will be computed at the time this static dictionary is constructed.  Let's say you launch the program at 11PM - DateTime.Today will be day 1, and it won't change when day 2 rolls around.

Instead, you should use a lambda / delegate here.  `() => return DateTime.Today` I believe would solve this problem

Reply:
That't a good point! I think I can solve that with a `Func<DateTime>` for the dictionary value